demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -148,77 +150,103 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
                 <>
                     {saveConflict.dialogs}
                     <MainContent>
-                        <TextField
-                            required
-                            fullWidth
-                            name="title"
-                            label={<FormattedMessage id="product.title" defaultMessage="Titel" />}
-                            validate={validateTitle}
-                        />
-
-                        <TextField required fullWidth name="slug" label={<FormattedMessage id="product.slug" defaultMessage="Slug" />} />
-
-                        <Field
-                            readOnly
-                            disabled
-                            endAdornment={
-                                <InputAdornment position="end">
-                                    <Lock />
-                                </InputAdornment>
+                        <FieldSet
```

Comment:
imho the config type (`"accordion"`) should match the used react component (`<FieldSet`).

@johnnyomair what do you think?

Reply:
I agree. Also, field set would be a more logical wording in the form context .